README.md

```diff
@@ -0,0 +1,289 @@
+|                                    |                   | [**SetUser**](https://www.onvif.org/ver10/device/wsdl/devicemgmt.wsdl#op.SetUser)                                               | Object           |
+| Metadata Configuration             | Media             | [GetMetadataConfiguration](https://www.onvif.org/ver10/media/wsdl/media.wsdl#op.GetMetadataConfiguration)                       | Object           |
+|                                    |                   | [GetMetadataConfigurations](https://www.onvif.org/ver10/media/wsdl/media.wsdl#op.GetMetadataConfigurations)                     | Object           |
+|                                    |                   | [GetCompatibleMetadataConfigurations](https://www.onvif.org/ver10/media/wsdl/media.wsdl#op.GetCompatibleMetadataConfigurations) | Object           |
+|                                    |                   | [**GetMetadataConfigurationOptions**](https://www.onvif.org/ver10/media/wsdl/media.wsdl#op.GetMetadataConfigurationOptions)     | Object           |
+|                                    |                   | [AddMetadataConfiguration](https://www.onvif.org/ver10/media/wsdl/media.wsdl#op.AddMetadataConfiguration)                       | Object           |
+|                                    |                   | [RemoveMetadataConfiguration](https://www.onvif.org/ver10/media/wsdl/media.wsdl#op.RemoveMetadataConfiguration)                 | Object           |
+|                                    |                   | [**SetMetadataConfiguration**](https://www.onvif.org/ver10/media/wsdl/media.wsdl#op.SetMetadataConfiguration)                   | Object           |
+| Video Streaming                    | Media             | [**GetProfiles**](https://www.onvif.org/ver10/media/wsdl/media.wsdl#op.GetProfiles)                                             | Object           |
+|                                    |                   | [**GetStreamUri**](https://www.onvif.org/ver10/media/wsdl/media.wsdl#op.GetStreamUri)                                           | Object           |
+| VideoEncoder  Config               | Media             | [GetVideoEncoderConfiguration](https://www.onvif.org/ver10/media/wsdl/media.wsdl#op.GetVideoEncoderConfiguration)               | Object           |
+|                                    |                   | [**SetVideoEncoderConfiguration**](https://www.onvif.org/ver10/media/wsdl/media.wsdl#op.SetVideoEncoderConfiguration)           | Object           |
+|                                    |                   | [GetVideoEncoderConfigurationOptions](https://www.onvif.org/ver10/media/wsdl/media.wsdl#op.GetVideoEncoderConfigurationOptions) | Object           |
+|                                    |                   | [GetNode](https://www.onvif.org/ver10/media/wsdl/media.wsdl#op.GetNode)                                                         | Object           |
```

Comment:
Link should be http://www.onvif.org/onvif/ver20/ptz/wsdl/ptz.wsdl#op.GetNode

Reply:
Fixed the wrong link.